.github/workflows/test-packer-qemu.yml

```diff
@@ -0,0 +1,49 @@
+name: Packer build qemu
+
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: codespaces
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+
+      - name: Setup packer
+        uses: hashicorp/setup-packer@1aa358be5cf73883762b302a3a03abd66e75b232 # v3.1.0
+        id: setup
+        with:
+          version: 1.11.2
+
+      - name: Generate cloud-init iso
+        run: |
+          sudo apt install genisoimage
+          echo '#cloud-config
+          chpasswd:
+            expire: False
+            users:
+            - {name: debian, password: password, type: text}
```

Comment:
@NichArchA82 Does this passwd get disabled later? or is it going to be the default for every VM via libvirt?

Reply:
when cloud-init runs again, it will run passwd - l debain essentially locking the debian account removing this temp password